src/views/vmd-home.view.ts

```diff
@@ -35,7 +35,15 @@ export class VmdHomeView extends LitElement {
       } else {
         const commune = event.detail.commune
         const departements = await State.current.departementsDisponibles()
-        const departement = departements.find(({ code_departement }) => code_departement === commune.codeDepartement)
+        if(!departements.find(d => d.code_departement==='om')) {
```

Comment:
It's not better to add this behavior in the `State.current.departementsDisponibles()` method?

Reply:
Oui, je suis d'accord

Après, ce qui serait encore mieux, c'est qu'on ait pas du tout à le faire :-D (eg, que le back le rajoute :) )